controllers/iochaos/types.go

```diff
@@ -27,35 +31,47 @@ import (
 	"github.com/pingcap/chaos-mesh/controllers/twophase"
 )
 
+type IochaosReqest struct {
```

Comment:
s/IochaosReqest/IOChaosRequest?

Reply:
I mean to find a place to hold the event recorder. It has to be accessed along the reconciler. If I put it into the reconciler as a field, it may be duplicate since each time when we create a reconciler, we have to put it into its constructor. I suggest to wrap a fresh request for each chaos scheduler and put it into it as a field. It's still a draft. Any suggestions ? @zhouqiang-cl

Reply:
OK. Will refactor it.

Reply:
If I wrap the `ctrl.Request` with a `IOChaosRequest`, it will lead to a few changes to `Reconcile` signature. Is that change affordable? @zhouqiang-cl

Reply:
If that is not affordable, I could put the event recorder into each reconciler as a filed. However, it is a little repeated. I think it tastes bad. Any better suggestions ?

Reply:
@cwen0 PTAL

Reply:
Can we need to define the other struct like `NetwrokReqest`/`PodRequest`?

Reply:
Yes. This is just a refine example. I will refine them later today.